src/builder.rs

```diff
@@ -396,6 +396,17 @@ impl ThingBuilder {
         self
     }
 
+    /// Set multi-language descriptions
+    pub fn descriptions<F>(mut self, f: F) -> Self
+    where
+        F: FnOnce(&mut MultiLanguageBuilder<String>) -> &mut MultiLanguageBuilder<String>,
+    {
+        let mut builder = MultiLanguageBuilder::default();
+        f(&mut builder);
+        self.descriptions = Some(builder.values);
+        self
+    }
+
```

Comment:
You are right about the missing method, but here we should implement the `HumanReadableInfo` instead (obviously fixing things for the other methods)

Reply:
I opened an issue about it, moving to HumanReadableInfo is an improvement overall :) Adding the method as-is was quicker.

Reply:
Ok, just remember that it is a breaking change -- if you need to use `descriptions` somewhere else, it will break as soon as we use the trait instead.

Reply:
We can break as much as needed since we aren't even to 0.1. Ideally once we have the preamble set it should not be noticeable.